clients/admin-ui/cypress/e2e/config-wizard.cy.ts

```diff
@@ -8,46 +8,6 @@ describe("Config Wizard", () => {
     }).as("getOrganization");
   });
 
-  // TODO: Update Cypress test to reflect the nav bar 2.0
-  describe.skip("Organization setup", () => {
```

Comment:
This is no longer accessible after the changes to the Add Systems flow last near, so deleting this legacy test entirely.